app/javascript/app/providers/sections-content-provider/sections-content-provider-actions.js

```diff
@@ -0,0 +1,42 @@
+import { createAction, createThunkAction } from 'redux-tools';
+import { INDOAPI } from 'services/api';
+
+export const fetchSectionsContentInit = createAction(
+  'fetchSectionsContentInit'
+);
+export const fetchSectionsContentReady = createAction(
+  'fetchSectionsContentReady'
+);
+export const fetchSectionsContentFail = createAction(
+  'fetchSectionsContentFail'
+);
+
+export const fetchSectionsContent = createThunkAction(
+  'fetchSectionsContent',
+  params => (dispatch, state) => {
+    const { SectionsContent } = state();
+    if (SectionsContent && SectionsContent.data && !SectionsContent.loading) {
```

Comment:
We might want to prevent fetching if we already have the content

Reply:
I got confused, I thought you meant if English content was already fetched, you switched to Indonesian and then again switched to English that that should be preserved. If that's not what you meant but just to prevent fetching when you switch from English to English or from Indonesian to Indonesian, i.e. the params don't change, then it's already prevented: https://github.com/ClimateWatch-Vizzuality/indonesia-platform/pull/17/files#diff-e69ae4e00b367d54a078f87e2c18cc61R15